### DISCUSSION TITLE
Question about the BG receive delay

### DISCUSSION THREAD

**Discussion by berryallday** [NightscoutFoundation/xDrip#3805] [Q&A]
![Screenshot_20241209_125652_xDrip+](https://github.com/user-attachments/assets/28d6ae4f-5aaa-448c-ac80-5b3096787f6d)

 it's about the x drip's BG receiving delay...i've keep searched internet and medical supervisor(an expert with insulin pump and other cgm or etc related with type 1 dm)

Im using medtronic minimed 780G pump with CGM included. connection between carelink and xdrip. and connect with my son

when i first using xdrip with receiving data via carelink it didn't have any BG delay. but ,
at some point suddenly the BG receiving delay kepts The cycle just becomes slightly longer or shorter and does not return to its original state.

 Does anyone knows the way how to deal with this problem? 

If anyone  could tell me about this, I would be so grateful. thanks a lot for anyone who concerned aboutIf you could tell me about this, I would be so grateful. thanks.

**Comment by Navid200** [comment]
Since you have cropped out the top portion of your screenshot, which shows which version of xDrip you are using, there is no way to know.
If you are using xDrip from a different repository or if you are using an old version of xDrip, it would make no sense to speculate what the cause of the problem may be.
Can you please always include the xDrip logo at the top of the system status page screenshot?

**Reply by berryallday** [reply]
![Screenshot_20241209_131041_xDrip+](https://github.com/user-attachments/assets/84f3524d-2676-459d-a273-3020e7654e88)

thanks

**Comment by Navid200** [comment]
@benceszasz I wonder if you can provide advice.  Thanks
